Makefile

```diff
@@ -215,22 +215,29 @@ SWARM_HOSTS            = $(shell docker node ls --format="{{.Hostname}}" 2>$(if
 docker-compose-configs = $(wildcard services/docker-compose*.yml)
 CPU_COUNT = $(shell cat /proc/cpuinfo | grep processor | wc -l )
 
+# NOTE: fore details on below SEE
+# https://github.com/docker/compose/issues/7771#issuecomment-765243575
+# below sed operation fixes above issue
+# `sed -E "s/cpus: ([0-9\\.]+)/cpus: '\\1'/"`
+# remove when this isssue is fixed, this will most likely occur
+# when upgrading the version of docker-compse
+
 .stack-simcore-development.yml: .env $(docker-compose-configs)
 	# Creating config for stack with 'local/{service}:development' to $@
 	@export DOCKER_REGISTRY=local \
 	export DOCKER_IMAGE_TAG=development; \
 	export DEV_PC_CPU_COUNT=${CPU_COUNT}; \
-	docker-compose --env-file .env --file services/docker-compose.yml --file services/docker-compose.local.yml --file services/docker-compose.devel.yml --log-level=ERROR config > $@
+	docker-compose --env-file .env --file services/docker-compose.yml --file services/docker-compose.local.yml --file services/docker-compose.devel.yml --log-level=ERROR config | sed -E "s/cpus: ([0-9\\.]+)/cpus: '\\1'/" > $@
```

Comment:
:-1: for docker-compose...
but could you use long option names? and also break the lines... it is too large for a 4K screen...

Reply:
yep .. adding line breaks will definitively help readability :-))